amaze

```diff
@@ -47,7 +47,7 @@ def amaze(ini_file):
 
     # see if we're using GW observations
     #using simulated observations currently only supported by KDE inference
-    #gwobs = True if args.model0=='gwobs' else False
+    gwobs = True if settings['true-model']==None else False
```

Comment:
Note that I removed "gwobs" and just do these parts of the code when settings['true-model']==None. So we can remove this line.